web/src/providers/Test/Test.provider.tsx

```diff
@@ -30,13 +31,15 @@ export const Context = createContext<IContext>({
 
 interface IProps {
   testId: string;
-  version?: number;
   children: React.ReactNode;
 }
 
 export const useTest = () => useContext(Context);
 
-const TestProvider = ({children, testId, version = 0}: IProps) => {
+const TestProvider = ({children, testId}: IProps) => {
+  const {run} = useTestRun();
+  const version = run.testVersion || 0;
```

Comment:
What's the advantage of this change? I've seen that we use the TestProvider outside of the test run context, I wouldn't like to have this coupled